### DISCUSSION TITLE
Wrong video after decoding using vvdec

### DISCUSSION THREAD

**Discussion by quchanhub** [fraunhoferhhi/vvenc#135] [Q&A]
Hi, I tried the following commands to encode and decode a YUV file using the latest version of VVenc and VVdec but I get a very strange decoded video, which is totally different from the input video (see the attached image). Do you know what is wrong?

./vvencapp --preset medium -i bus_cif.yuv -s 352x288 -o out.266
./vvdecapp -b out.266 -o bus_recon.yuv

![bus](https://user-images.githubusercontent.com/21310047/153114188-97e9bf05-cf7f-45a3-b43a-b34df0af1bf0.png)

**Comment by adamjw24** [comment]
Please have a look at: https://github.com/fraunhoferhhi/vvenc/issues/26 and https://github.com/fraunhoferhhi/vvdec/issues/4
